drivers/cc2420/cc2420.c

```diff
@@ -205,13 +205,9 @@ int cc2420_rx(cc2420_t *dev, uint8_t *buf, size_t max_len, void *info)
 
         /* fetch and check if CRC_OK bit (MSB) is set */
         cc2420_fifo_read(dev, &crc_corr, 1);
-        if (!(crc_corr & 0x80)) {
-            DEBUG("cc2420: recv: CRC_OK bit not set, dropping packet\n");
-            /* drop the corrupted frame from the RXFIFO */
-            len = 0;
-        }
```

Comment:
I don't like, that the dropping of the frame is removed completely. Shouldn't this be configurable?

Reply:
Yes, definetely! Where to define it, and how to name it? GNRC_NETIF_DROP_CORRUPT_PACKETS or something?

Reply:
> Where to define it,

A `NETOPT`, in `sys/include/net/netopt.h` might be the most generalized.

> and how to name it?

How about `NETOPT_CHECK_PKT_ERROR`? This way it is independent from error checking via checksums.

Reply:
`gnrc_netif` would not be correct since this is a device driver thing ;-), not a network interface and certainly not a GNRC feature.